lib/src/doc/lives.rs

```diff
@@ -103,11 +100,20 @@ impl<'a> Document<'a> {
 				if stm.is_delete() {
 					// Send a DELETE notification
 					if opt.id()? == lv.node.0 {
-						let thing = (*rid).clone();
 						chn.send(Notification {
-							id: lv.id.clone(),
+							id: lv.id,
 							action: Action::Delete,
-							result: Value::Thing(thing),
```

Comment:
This is a breaking change. Clients (JS in example) previously expected a string, but now receives an object with the entire record.

I like the change btw, but it's breaking unfortunately